spec/models/product_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe Product, type: :model do
+  let!(:product){ FactoryBot.create(:product) }
```

Comment:
phân biệt let và let!

Reply:
- let! sẽ tạo mới một đối tượng mỗi khi một test case được thực hiện và 1 đối tượng được tạo bởi let! chỉ tồn tại trong khối do end của it

- let sẽ chỉ tạo đối tượng nếu test case có yêu cầu gọi đến một đối tượng cụ thể